Tests/MongoSwiftTests/CommandMonitoringTests.swift

```diff
@@ -182,38 +176,38 @@ private struct CMTest {
             let options = FindOptions(batchSize: batchSize,
                                         comment: modifiers?["$comment"] as? String,
                                         hint: hint,
-                                        limit: self.args["limit"] as? Int64,
+                                        limit: self.op.args["limit"] as? Int64,
                                         max: modifiers?["$max"] as? Document,
                                         maxTimeMS: maxTime,
                                         min: modifiers?["$min"] as? Document,
                                         returnKey: modifiers?["$returnKey"] as? Bool,
                                         showRecordId: modifiers?["$showDiskLoc"] as? Bool,
-                                        skip: self.args["skip"] as? Int64,
-                                        sort: self.args["sort"] as? Document)
+                                        skip: self.op.args["skip"] as? Int64,
+                                        sort: self.op.args["sort"] as? Document)
 
             // we have to iterate the cursor to make the command execute
-            do { for _ in try collection.find(filter ?? [:], options: options) {} } catch { }
+            for _ in try! collection.find(filter, options: options) {}
```

Comment:
Is full iteration necessary because the tests expect both an initial `find` and `getMore` command(s)? If not, would it make sense to only advance once to ensure that the initial `find` is executed and avoid anything beyond that?

Reply:
yup, for example [this](https://github.com/mongodb/specifications/blob/4f212acb5154a3c3d124525ad1f0f308efa93ef8/source/command-monitoring/tests/find.json#L170) test case expects a `getMore`.